MarvellOS.py

```diff
@@ -49,11 +64,11 @@ def tick():
     if time2 != time1:
         time1 = time2
         clock.config(text=day + "      " + time2 + "  ")
-    if truth == True:
-        try:
-            AFTER = welcome_screen.after(200, tick)
-        except:
-            welcome_screen.after_cancel(AFTER)
+    if truth:
+        #       try:
+        AFTER = welcome_screen.after(200, tick)
+    #       except:
```

Comment:
You don't have to comment out the except block entirely. Just find the error that it is trying to except, and except that.
If you cannot find that, last resort can be to use at least `except Exception:`.